src/RandomUser.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import Button from "./Button";
+import UserProfile from "./UserProfile";
+
+const RandomUser = () => {
+  const [user, setUser] = useState({});
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const fectchUser = () => {
+    fetch(`https:randomuser.me/api/?results=5`);
+    setLoading(true)
+      .then((res) => {
+        return res.json();
+      })
+      .then((response) => {
+        setLoading(false);
+        setError(false);
+        setUser({
+          first: response.results[0].name.first,
+          last: response.results[0].name.last,
+          city: response.results[0].location.city,
+          country: response.results[0].location.country,
+          phoneNumber: response.results[0].phone,
+        });
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
You do not want to log the error to the console. This is not visible to the user.